entries/o/o2switch.fr.json

```
@@ -0,0 +1,14 @@
+{
+    "o2switch": {
+        "domain":"o2switch.fr",
+        "url": "https://clients.o2switch.fr",
+        "tfa": [
+            "email"
+        ],
+        "keywords": [
+            "hosting",
+            "domains"
+        ],
+        "notes": "Email 2FA is always active, when logging in from a new device."

```

```suggestion
        "notes": "Email 2FA is always active, when logging in from a new device.",
        "regions": [
            "fr"
        ]
```

They appear to only be available in France